src/modules/todo.js

```diff
@@ -0,0 +1,7 @@
+export default class todo {
```

Comment:
- [ ] Classes in Javasript are capitalized, please consider capitalizing `todo` class, I have checed the whole code and didnt find where you are creating a new `todo` object (eg. `const task = new todo(id, description, completed) `), which means that this class is constantly empty, subsequently the app does not work as expected. 

- [ ] Please consider creating an  object of this class to make the app work. 
- [ ] Currently,  tasks are not being recorded in the local storage because of errors I have mentioned. Once all these errors are fixed, please make sure that local storage is being updated, you can check it in google chrome `developer tools => application => local storage`